### DISCUSSION TITLE
列表模式代码注入管理

### DISCUSSION THREAD

**Discussion by wan92hen** [halo-dev/halo#5589] [Ideas]
### 描述一下此特性

目前的代码注入功能，支持在全局 head 标签、内容页 head 标签 及全局页脚位置注入自定义代码，当注入的代码类型较多时，在同一个编辑器中进行管理不是很方便。

期望能够以数据列表形式来管理要注入的代码，一条代码注入数据可以包含以下信息：
- 名称
- 描述
- 代码内容
- 代码位置
- 需要注入的页面
- 是否启用

**Comment by ruibaby** [comment]
似乎很适合用插件实现，类似于 Flarum 的 `HTML Head Items`  插件：

<img width="1069" alt="image" src="https://github.com/halo-dev/halo/assets/21301288/91a1411d-2b87-4f9e-850f-55cc07b49709">

如果要将目前的代码注入改为这种方式的话，可能会是个破坏更新。
